docker/wait4/wait4.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  // format version sem-ver
+  // `v{major}.${minor}.${patch}`
+  wait4: 'v0.1.0',
+
+  // How many times should we retry waiting for a service?
+  retries: 5,
+
+  // How many ms to wait before retrying a service connection?
+  waitMs: 2500,
+
+  // services definitions
+  services: [
+    {
+      name: 'als-consent-oracle',
+      wait4: [
+        {
+          description: 'MySQL Consent Oracle',
+          uri: 'oracle-mysql:3306',
+          method: 'mysql',
+          // customized RC setup
```

Comment:
if we drop RC config there will be a need to refactor RC based wait4 methodMySQL strategy for knex or use `ncat` strategy